server/neptune/workflows/internal/activities/terraform_test.go

```diff
@@ -28,7 +28,11 @@ func (t *testStreamHandler) Stream(jobID string, msg string) {
 
 }
 
-func (t *testStreamHandler) Close(ctx context.Context, jobID string) error {
+func (t *testStreamHandler) CloseJob(ctx context.Context, jobID string) error {
+	return nil
+}
+
+func (t *testStreamHandler) CleanUp(ctx context.Context) error {
 	return nil
 }
```

Comment:
what is this doing? Shouldn't this be tested? if it's not used, then the interface shouldn't have these methods.

Reply:
Removed the unused methods!